Leaf API.postman_collection.json

```
@@ -4513,6 +4513,142 @@
 									"response": []
 								}
 							]
+						},
+						{
+							"name": "Prescription",
+							"item": [
+								{
+									"name": "RavenSlingshot",
+									"item": [
+										{
+											"name": "List all prescription",
```

```suggestion
											"name": "Get RavenSlingshot prescriptions",
```
I noticed we use `Get` instead of list in most places. Just nitpicking for consistency sake, but either way is fine.

Also, for the leaf user provider endpoints, each one of them has the provider in the title. This could be done here too.

---

These requests are separated into folders for every provider, but I agree we could specify even more and add on their title too.

---

Yep, "Get" is better than "List" :)

---

f1af451